runtime/nls/j9vm/j9vm.nls

```diff
@@ -2395,13 +2395,6 @@ J9NLS_VM_DEPRECATED_OPTION.system_action=The JVM will print a deprecation warnin
 J9NLS_VM_DEPRECATED_OPTION.user_response=Remove the command line option from the command line.
 # END NON-TRANSLATABLE
 
-J9NLS_VM_CRIU_CHECK_TRANSITION_TO_DEBUG_INTERPRETER_FAILED=The check for the transition to the debug interpreter failed
```

Comment:
I dont think we should modfy the .nls files. These need to line up for backwards compatibility. @JasonFengJ9 can correct me if im wrong.

Reply:
> I dont think we should modfy the .nls files. These need to line up for backwards compatibility. 

That's correct.

https://github.com/eclipse-openj9/openj9/blob/ebe8718d5838fc63d1ebeda5b1af37e2885ef320/runtime/nls/j9vm/j9vm.nls#L27-L29

Reply:
Ah. Perfect. Now I know. Thanks